docs/content/development/extensions.md

```diff
@@ -58,6 +58,7 @@ Core extensions are maintained by Druid committers.
 |druid-parquet-extensions|Support for data in Apache Parquet data format. Requires druid-avro-extensions to be loaded.|[link](../development/extensions-core/parquet.html)|
 |druid-protobuf-extensions| Support for data in Protobuf data format.|[link](../development/extensions-core/protobuf.html)|
 |druid-s3-extensions|Interfacing with data in AWS S3, and using S3 as deep storage.|[link](../development/extensions-core/s3.html)|
+|druid-ec2-extensions|Interfacing with AWS EC2 for autoscaling middle managers|UNDOCUMENTED|
```

Comment:
This is I guess sort of documented at `../../configuration/index.html#autoscaler`, but I think it would probably be better in the long run to make a doc for this extension and link those autoscaler docs to it. We should also mention in the existing autoscaler docs in the giant configuration page that the `druid-ec2-extensions` need to be loaded for it to work, but I don't really consider either of these blockers for this PR to get merged since it isn't documented super well in the first place.